spec/models/team_spec.rb

```diff
@@ -12,7 +12,34 @@
 
 RSpec.describe Team, type: :model do
   it "is valid when produced by a factory" do
-    team = FactoryBot.build(:team)
+    team = FactoryBot.create :team
```

Comment:
Is there a reason you need to use `create` and make the team instance persistent? Using `build` would speed up the tests I think...

Reply:
I do not know why I replaced the function - changed back to build.